site/develop/index.md

```diff
@@ -67,6 +67,26 @@ helps to agree on the general approach. Log a
 [JIRA case](https://issues.apache.org/jira/browse/CALCITE) for your
 proposed feature or start a discussion on the dev list.
 
+Before opening up a new JIRA case, have a look in the existing issues.
+The feature or bug that you plan to work on may already be there.  
+
+If you are going to take on the issue right away assign it to your self.
+To assign issues to your self you have to be registered as a contributor 
+to the project. In order to do that, send an email to the developers list
+and provide your JIRA username.   
+
+If you think the issue can make it to the ongoing release set the fix version
```

Comment:
"If you are committed to fixing the issue before the upcoming release"

Reply:
Done